onboarding.md

```diff
@@ -0,0 +1,35 @@
+# Developers Onboarding
+
+## Welcome in to our developers hub 
+
+We look forward to working together!
+
+## Start Here 
+
+**What is Cambiatus?**
+
+Open online platform that enables communities/groups with environmental and social objectives to create and implement their own community currencies. 
+
+At Cambiatus we make the blockchain technology available and user friendly, so these communities can take full advantage of it and solve their environmental or social problems. Cambiatus is also becoming a tool to make DAC (decentralized autonomous communities) creation easy and fun. 
+
+**What is DAC?**
+
+We have put together a document called the [Path to Knowledge](https://docs.google.com/document/d/1cKmTSR9VpPkYyD-uhpnnAuF4JITGWSu6X5BP9le1yMw/edit) in which we offer some sources of information which you can follow to learn about all the above topics. 
+
+**Now what really means Cambiatus?**
+
+In Spanish “Cambio” or "Cambiar" is the word used to express “Change”.  Cambiatus refers to a change in our mindset, a change in the way we understand money, organizations and collaboration. 
+
+When people start using Cambiatus methodologies and tools, something changes inside them, they get it! 
```

Comment:
I think we can add a section here linking to the Features pages

Reply:
Sorry got confused. What features page and what kind of section?